api-controllers/routes/admin.js

```diff
@@ -71,6 +71,9 @@ router.route('/service-fabrik/maintenance')
   .put(controller.handler('updateMaintenance'))
   .get(controller.handler('getMaintenance'))
   .all(commonMiddleware.methodNotAllowed(['GET', 'POST', 'PUT']));
+router.route('/update/schedules')
```

Comment:
I put it under /instances as this can later on be extended to non-BOSH based instances as well and all the underlying methods that we use are also using instances in the naming conventions.